spec/checker_jobs/notifiers/email_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "support/email_helpers"
+
+RSpec.describe CheckerJobs::Notifiers::Email, :configuration, :email do
+  include EmailHelpers
+
+  let(:instance) do
+    CheckerJobs::Checks::EnsureFewer.new(
+      checker_klass, "ensure_name", { than: 3 }, Proc.new { 5 }
+    )
+  end
+
+  let(:checker_klass) do
+    Class.new do
+      include CheckerJobs::Base
+      notify :email, to: "oss@drivy.com"
+    end
+  end
+
+  describe "with default" do
+    subject(:perform) { instance.perform }
+
+    it "sends an email" do
+      sends_an_email
+    end
+  end
+
+  describe "with wrong params" do
+    subject(:perform) { instance.perform }
+
+    let(:checker_klass) do
+      Class.new do
+        include CheckerJobs::Base
+        notify :email, to: 42
+      end
+    end
```

Comment:
On the `master` branch I tried to move those classes in contexts, quite similar to what you did on `drivy-tools`. The idea is to have contexts to include into the specs in order to get ready to use classes matching a given configuration.

You can have a look at it here https://github.com/drivy/checker_jobs/commit/26204755f1770e5dc412b362327f1f5767d3ba66 feedbacks on the approach are welcome!